task/04-arrays-tasks.js

```diff
@@ -337,7 +390,12 @@ function sortDigitNamesByNumericOrder(arr) {
  *   [ 1, 10, 100, 1000 ]  => 1111
  */
 function getItemsSum(arr) {
-   throw new Error('Not implemented');
+    
+   if(arr.length === 0)
+      return 0;
```

Comment:
redundant logic. Please, just use initial value in reduce.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce